README.md

```diff
@@ -39,6 +39,10 @@ Related materials:
 
 Difficulty: hard
 
+## Preparation
+
+After downloading and extracting the files, open the `kaggle.json` file (located in the root directory). Replace the <> placeholders with your own username-API key pair, and then save the changes. To obtain a Kaggle API key, log in to your [Kaggle](https://www.kaggle.com/) account and in the settings menu, generate a new kaggle.json file containing the API token. Replace the values in the `kaggle.json` file.
+
```

Comment:
`Preparation` could be a subchapter of `Installation`.

Reply:
Also list some caveats if there are any! Like ML model stuck `here`, or training is not ready yet. If there is a Python Notebook, which works, upload it into the `notebooks` directory and document its usage in `README.md`.

Reply:
Installation subchapter Preparation, add notebook and edit README with executing "how-to"